src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1451,70 +1455,31 @@ private void CheckMemberNameConflicts(DiagnosticBag diagnostics)
                     // second and third categories as follows:
 
                     var conversion = symbol as SourceUserDefinedConversionSymbol;
```

Comment:
(I know you didn't write this line of code but) is there a test that can accomplish this semantically without depending on the specific implementation hierarchy?